modules/ROOT/pages/autocompleter.adoc

```diff
@@ -23,7 +23,11 @@ The two arguments this method take are:
 [cols="1,2,1,4",options="header"]
 |===
 |Name |Value |Requirement |Description
-|ch |string (of one character) |Required |The character to trigger the autocompleter.
+|ch |`+string+` (of one character) |Required |The character to trigger the autocompleter.
+include::partial$misc/admon-deprecated-6.2v.adoc[]
+It has been replaced with the `+trigger+` option, which should be used instead.
```

Comment:
Do we explain why we deprecate it anywhere? I assume in this case it's because autocompleters can now have triggers that are more than one character, but do we need to say that? Do we need to provide the "no-op" migration advice? Or is that just something we cover in Release Notes?

Reply:
It's something that would normally go in the release notes if necessary from what I recall, as it ends up being too wordy to put here 🤷

Reply:
I can confirm that this is in the in progress release notes @brian-forte